src/main/java/com/github/awwkoala/selenium/util/WaitUtils.java

```diff
@@ -1,24 +1,39 @@
 package com.github.awwkoala.selenium.util;
 
+import org.openqa.selenium.NoSuchElementException;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.FluentWait;
 import org.openqa.selenium.support.ui.WebDriverWait;
 
+import java.time.Duration;
+
 public class WaitUtils {
 
   private final WebDriverWait wait;
+  private final FluentWait<WebDriver> fluentWait;
+  private final int timeOutInSec = 5;
```

Comment:
Same thing with longer variable names, as in previous PR
Or you can ever just inline them, they are used in one place anyway